pkg/apis/ceph.rook.io/v1/network.go

```diff
@@ -32,9 +32,14 @@ func (n *NetworkSpec) IsMultus() bool {
 	return n.Provider == NetworkProviderMultus
 }
 
-// IsHost get whether to use host network provider. This method also preserve
-// compatibility with the old HostNetwork field.
-func (n *NetworkSpec) IsHost() bool {
+// IsHost is intended to be used to determine if the rook operator should configure
+// managed pods to use host networking.
+// This behavior is enabled by configuring the cephCluster with the "host" network provider.
+// This method also maintains compatibility with the old HostNetwork setting
+// which is incompatible with other network providers: HostNetwork set to true
+// together with an empty or unset network provider has the same effect as
+// network.Provider set to "host"
+func (n *NetworkSpec) IsHost() (ret bool) {
```

Comment:
`ret` isn't referenced so it can be removed
```suggestion
func (n *NetworkSpec) IsHost() bool {
```